package.json

```diff
@@ -3,14 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
+    "compile": "rm -rf bin lib && ./node_modules/.bin/babel -s -d lib src/lib && ./node_modules/.bin/babel -s -d bin src/bin",
+    "install": "npm run compile",
+    "pretest": "npm run compile",
     "test": "test/runtests.sh"
   },
+  "babel": {
+    "sourceMaps": true,
+    "optional": [
+      "runtime"
+    ],
+    "stage": 1
+  },
```

Comment:
Okay, sounds good... As discussed in the other PR maybe in the future this is something we can load from an npm module...